Gir.toml

```diff
@@ -612,6 +612,45 @@ status = "generate"
     ignore = true
 
 [[object]]
+name = "Gtk.TreeModelFilter"
+status = "generate"
+    [[object.function]]
+    name = "clear_cache"
+    [[object.function]]
+    name = "get_model"
```

Comment:
For other functions as well, I wanted you to confirm if the `Option` wrapping was needed.